integration-tests/tests.ts

```diff
@@ -1,30 +1,60 @@
-import {
-  test,
-  expect,
-  ConsoleMessage,
-  Page,
-  TestInfo,
-  BrowserContext,
-} from "@playwright/test";
+import { test, expect, ConsoleMessage, Page, TestInfo } from "@playwright/test";
 import fs from "fs";
 
-const BASE_URL = process.env.BASE_URL;
-const BWD_BASE_URL = process.env.BWD_BASE_URL;
-const options = {
-  baseURL: BASE_URL,
-  bwdBaseURL: BWD_BASE_URL,
-};
-test.use(options);
+import {
+  initMessages,
+  saveMessage,
+  getMessages,
+  clearMessages,
+} from "./messages";
+import {
+  canvasUrl,
+  awaitAnalysisLoaded,
+  awaitAnalysis,
+  bwdUrl,
+  caretPos,
+  createEmptyHTTPHandler,
+  createHTTPHandler,
+  createRepl,
+  createWorkerHandler,
+  expectContainsText,
+  expectExactText,
+  getElementSelectionStart,
+  getStyleProperty,
+  gotoAST,
+  gotoHash,
+  Locators,
+  get,
+  post,
+  pressShortcut,
+  selectAll,
+  waitForEmptyEntryBox,
+} from "./utils";
```

Comment:
I'm not a big fan of so many imports from one file.
I thought about breaking down `utils` but spent too long on _naming things_ so gave up. Open to suggestions.

Reply:
Fine by me